requirements-base.in

```diff
@@ -1,12 +1,13 @@
 # Base dependencies. Don't edit or override this file.
 # Use requirements.in to add integration-specific dependencies instead.
 environs~=9.5.0
-pydantic~=1.10.13
+pydantic~=1.10.15
 fastapi~=0.103.2
 uvicorn~=0.23.2
-gundi-core~=1.3.0
-gundi-client-v2~=2.2.0
+gundi-core~=1.5.0
+gundi-client-v2~=2.3.1
```

Comment:
since the base requirements changed, we need to re-generate the `requirements.txt` too